### PR TITLE
[SDPAP-6973]hook update to delete page content type and existing nodes

### PR DESCRIPTION
**Pull request by yeniatencio**
Jira: https://digital-vic.atlassian.net/browse/SDPAP-7015
Updated: 
1. Remove any existing nodes of type page content type.
2. Remove page content type.

### CODE REVIEW

File: baywatch.install [hunk 1]
```diff
@@ -40,3 +41,33 @@ function baywatch_install() {
   $baywatch->enable_bay_platform_dependencies();
   $baywatch->enable_tide_site_theming();
 }
+
+/**
+ * remove any existing nodes of type page content type
```
Comment (nicksantamaria) [inline baywatch.install:46]: I'm not at all across what this PR is about, but deleting every page node across all tide projects seems quite risky.

**Comment by krakerag** [comment]
@yeniatencio I can't see on the ticket any mention of deleting this page content type from _all_ cms being confirmed - only from royalcommission.

I can't pass this without someone taking responsibility for the decision of deleting content of this type across all CMS rather than just the royalcommission cms like the ticket says.

Let me know please :)

**Review by krakerag — CHANGES_REQUESTED** [review]
Can we find out if this is meant to be deleted from all CMS or just the royalcommission CMS please

**Comment by yeniatencio** [comment]
> Can we find out if this is meant to be deleted from all CMS or just the royalcommission CMS please

Hi @nicksantamaria @krakerag , I am following solution direction of this [ticket](https://digital-vic.atlassian.net/browse/SDPAP-7015) and it mentions to create a hook update in baywatch module to remove any existing node of the page content type and remove permanently page content type. 
Here is the screenshot of the subtasks tickets. 
Screenshot 2024-01-22 at 9 53 13 am
 
I am going to confirm with Vanessa if remove page content type from all cms is what we are aiming for.

**Comment by yeniatencio** [comment]
Vanessa and Sarah confirmed that Health is still using page so it shouldn't be removed in core. Closing this ticket.

**Pull request closed without merging**